tests/endtoend/test_eventhub_functions.py

```diff
@@ -89,8 +89,11 @@ def test_eventhub_trigger_with_metadata(self):
         self.assertIsNotNone(event['metadata'])
         metadata = event['metadata']
         sys_props = metadata['SystemProperties']
-        enqueued_time = parser.isoparse(metadata['EnqueuedTimeUtc'])
-        self.assertTrue(start_time < enqueued_time < end_time)
+        enqueued_time = parser.isoparse(metadata['EnqueuedTimeUtc']).astimezone(
+            tz=tz.UTC)
+
+        self.assertTrue(
+            start_time.timestamp() < enqueued_time.timestamp() < end_time.timestamp())  # NoQA
```

Comment:
This comparison was happening between timezone aware and timezone naive values, which caused the tests to break. Fixed this by explicitly adding the timezone awareness in `enqueued_time`.